### ISSUE TITLE
Browser window turns white in Min 1.29.0

### ISSUE THREAD

**Issue by PalmerAL** [minbrowser/min#2321]
I downloaded the latest release of the Min browser [min-1.29.0-amd64.deb](https://github.com/minbrowser/min/releases/download/v1.29.0/min-1.29.0-amd64.deb) just now. After installing on my Ubuntu system (Ubuntu 22.04.3 LTS x86_64), I opened the browser. On the browsers default landing page, clicking on the "Start Browsing" button and typing in a web address simply kills the browser, and the window turns white.

Restarting the browser doesn't help. Finally I had to uninstall this release, and switch to an older release [min-1.28.1-amd64.deb](https://github.com/minbrowser/min/releases/download/v1.28.1/min-1.28.1-amd64.deb). This older release seems to be working fine.

-------

NOTE : This is not a RAM issue since my system has 32 GB of RAM, CPU = 11th Gen Intel i5-11400 (12) @ 4.400GHz

_Originally posted by @aakashsinghbais in https://github.com/minbrowser/min/issues/267#issuecomment-1733382315_

**Comment by PalmerAL** [comment]
@aakashsinghbais

* Does launching Min with `--disable-hardware-acceleration` help?
* Are you able to open the preferences page and enable "use separate titlebar"? If so, when the window turns white, can you go to developer > inspect browser, and what do you see?

**Comment by aakashsinghbais** [comment]
Okay,
I launched `min-1.29.0` from terminal this time to check if any errors are logged and got the following:
![image](https://github.com/minbrowser/min/assets/56718090/a81c68f2-8f0b-45db-adbe-a4610160fd67)

- **Disabling Hardware Acceleration** : Using `--disable-hardware-acceleration` does not help.
- **Using Separate Title bar & Accessing Developer Options** : the white window of min is just a blank window with no title bar, it does not respond to any keys or mouse clicks, it is just a dead white rectangle that can be dragged around on the monitor and can be resized, that's all. Every time I toggle from this white window to any other window on my system, a new copy of the above shown error is logged on the terminal from which I ran min using the `--disable-hardware-acceleration` flag.

---

NOTE : The browser dies (turns into a dead white window) after a 4-5 second gap after initially opening fine as can be seen below:

[Screencast from 26-09-23 02:40:02 PM IST.webm](https://github.com/minbrowser/min/assets/56718090/be6d2072-984b-4f6e-929e-494fb8e86419)

**Comment by Sam0l1y** [comment]
I have same problem in elementary OS 7.1. I am using NVIDIA GeForce GTX 1660 SUPER.

**Comment by PalmerAL** [comment]
If you're having this issue, can you please try the following and see if it works?
```
git clone https://github.com/minbrowser/min.git
cd min
npm install
npm install electron@26.4.3
npm run start
```
I think this may be https://github.com/electron/electron/pull/40377, for which a fix was just released. I can't reproduce this though, so I can't tell for sure.

**Comment by BlueManCZ** [comment]
> npm install electron@26.4.3

I can confirm that upgrading Electron to 26.4.3 fixes the issue. :+1:

**Comment by PalmerAL** [comment]
The new release published today (1.30) should resolve this issue; please let me know if you're still seeing it after upgrading.
